src/client/Container/Container.ts

```diff
@@ -109,7 +141,7 @@ export class Container {
    * Use `.userDefinedFunctions` for creating new user defined functions, or querying/reading all user defined functions.
    * @param id The id of the {@link UserDefinedFunction}.
    */
-  public userDefinedFunction(id: string): UserDefinedFunction {
+  protected __userDefinedFunction(id: string): UserDefinedFunction {
```

Comment:
Why adding the double underscore prefix?

Reply:
single underscore is yelled at by tslint and I want a different name than the old type so we can still build our mocked types using the old api surface area.